pwa/src/templates/documentation/standardsDocumentation/StandardsDocumentationTemplate.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from "react";
+import * as styles from "./StandardsDocumentationTemplate.module.css";
+import { Container } from "@conduction/components";
+import {
+  Button,
+  Heading1,
+  Heading3,
+  Heading5,
+  LeadParagraph,
+  Link,
+  List,
+  ListItem,
+  Paragraph,
+} from "@gemeente-denhaag/components-react";
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "@gemeente-denhaag/table";
+import clsx from "clsx";
+import { TEMPORARY_STANDARDS } from "../../../data/standards";
+import { ExternalLinkIcon, ArrowRightIcon } from "@gemeente-denhaag/icons";
+import { navigate } from "gatsby";
+import { useTranslation } from "react-i18next";
+
+export const StandardsDocumentationTemplate: React.FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Container layoutClassName={styles.container}>
+      <section className={styles.section}>
+        <Heading1>Gebruikte standaarden</Heading1>
+        <LeadParagraph>De standaarden die gebruikt worden door Open Catalogi</LeadParagraph>
```

Comment:
I suggest making this a bit larger of a text ~~and to link to https://forumstandaardisatie.nl/ within that text~~.

Maybe add a button underneath to the [forumstandaardisatie](https://forumstandaardisatie.nl/) _and_ a button linking to https://forumstandaardisatie.nl/open-standaarden/verplicht 

![image](https://user-images.githubusercontent.com/66552932/191928933-5993e926-bf44-43e8-8b1e-db9292df6379.png)

Reply:
![image](https://user-images.githubusercontent.com/66552932/191927719-a9874e78-9a62-46f4-ba80-ea0c71d64bb9.png)